go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/buildpack/imgutil
```

Comment:
Given that this is a library, I don't see a strong motivation for vendoring all the dependencies. Could we rip those out?

Reply:
Sure, no problem

Reply:
Done.